components/brave_rewards/core/rewards_engine_helper.h

```diff
@@ -50,7 +50,7 @@ class WithHelperKey {
   static const void* GetHelperKey() { return &kHelperKey; }
 
  private:
-  static inline const int kHelperKey = 0;
+  static inline constexpr int kHelperKey = 0;
```

Comment:
yes, that's okay because `constexpr` constants are guaranteed to have internal linkage, however it being `inline` will also guarantee it has a single address.

Reply:
I think the `inline` specifier is implied by `constexpr`. (In the guidance doc `static inline constexpr` is labelled "Unecessarily verbose").

Reply:
Fixed to match the doc. Thanks.